Assets/MixedRealityToolkit.SDK/Inspectors/UX/Collections/ScrollingObjectCollectionInspector.cs

```diff
@@ -0,0 +1,340 @@
+                EditorGUILayout.Space();
+                EditorGUILayout.PropertyField(canScroll);
+                EditorGUILayout.PropertyField(setUpAtRuntime);
+                EditorGUILayout.PropertyField(useCameraPreRender);
+                EditorGUILayout.Space();
+            }
+
+            EditorGUILayout.Space();
+
+            if (GUILayout.Button("Update Collection"))
+            {
+                scrollContainer.UpdateCollection();
+                EditorUtility.SetDirty(scrollContainer);
+            }
+
+            EditorGUILayout.Space();
+
+
+            EditorGUILayout.LabelField("Scrolling Properties", EditorStyles.boldLabel);
+
+            using (new EditorGUI.IndentLevelScope())
+            {
+                EditorGUILayout.PropertyField(dragTimeThreshold);
+                EditorGUILayout.PropertyField(handDeltaMagThreshold);
+                EditorGUILayout.PropertyField(useNearScrollBoundary);
+                EditorGUILayout.Space();
+
+                EditorGUILayout.PropertyField(occlusionPositionPadding);
+                EditorGUILayout.PropertyField(occlusionScalePadding);
+                EditorGUILayout.Space();
+
+                EditorGUILayout.PropertyField(animationCurve);
+                EditorGUILayout.PropertyField(animationLength);
+            }
+
+            EditorGUILayout.Space();
+
+            EditorGUILayout.LabelField("Velocity Properties", EditorStyles.boldLabel);
+            using (new EditorGUI.IndentLevelScope())
+            {
+                EditorGUILayout.PropertyField(velocityType);
+
+                if (velocityType.enumValueIndex <= 1)
+                {
+                    EditorGUILayout.PropertyField(velocityMultiplier);
+                    EditorGUILayout.PropertyField(velocityFalloff);
+                    EditorGUILayout.PropertyField(bounceMultiplier);
+                }
+            }
+
+            EditorGUILayout.Space();
+
+            showUnityEvents = EditorGUILayout.Foldout(showUnityEvents, "Unity Events: ", true);
+
+            if (showUnityEvents)
+            {
+                using (new EditorGUI.IndentLevelScope())
+                {
+
+                    EditorGUILayout.PropertyField(clickEvent);
+                    EditorGUILayout.PropertyField(touchEvent);
+                    EditorGUILayout.PropertyField(untouchEvent);
+                    EditorGUILayout.PropertyField(momentumEvent);
+                }
+            }
+
+            EditorGUILayout.Space();
+
+            showOverrideButtons = EditorGUILayout.Foldout(showOverrideButtons, "Scrolling debug buttons", true);
+
+            EditorGUILayout.Space();
+
+            if (showOverrideButtons)
+            {
+                using (new EditorGUI.IndentLevelScope())
+                {
+                    animateTransition = EditorGUILayout.Toggle("Animate", animateTransition);
+
+                    EditorGUILayout.BeginHorizontal();
```

Comment:
You can use 
using (new EditorGUILayout.HorizontalScope())
{
}

Everywhere you do begin / end horizontal